samples/matter/common/src/bridge/bridge_storage_manager.cpp

```diff
@@ -123,31 +189,109 @@ bool BridgeStorageManager::RemoveBridgedDeviceType(uint8_t bridgedDeviceIndex)
 	return Nrf::GetPersistentStorage().NonSecureRemove(&id);
 }
 
-bool BridgeStorageManager::StoreBridgedDevice(const MatterBridgedDevice *device, uint8_t index)
+bool BridgeStorageManager::LoadBridgedDevice(BridgedDevice &device, uint8_t index)
 {
-	if (!device) {
+	Nrf::PersistentStorageNode id = CreateIndexNode(index, &mBridgedDevice);
+	size_t readSize = 0;
+	uint8_t buffer[sizeof(BridgedDevice) + kMaxUserDataSize];
+	uint16_t counter = 0;
+	const uint8_t mandatoryItemsSize = sizeof(device.mEndpointId) + sizeof(device.mDeviceType) + sizeof(device.mNodeLabelLength) + sizeof(device.mUserDataSize);
```

Comment:
`mUserDataSize` is not a mandatory field, it seems.

Reply:
Well, even if there is no user data we have to save it to settings with size equal to 0, because we don't have other way to decide whether data are present or not after loading, right?

Reply:
Oh wait, what I wrote doesn't match the implementation :D I have to re-think this.

Reply:
Ok, you are right, it's not needed.